.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -1,5 +1,5 @@
 blank_issues_enabled: false
 contact_links:
   - name: ❓ Ask a question
-    url: https://github.com/WalletConnect/walletconnect-monorepo/discussions/new?category=web3modal
```

Comment:
No categories ?

Reply:
@lukaisailovic can we replace web3modal category with appkit on the monorepo issues ? 🙏

Reply:
Yes. Lets do it.